c/datasetHeartbeatService.c

```diff
@@ -0,0 +1,110 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+
+#ifdef METTLE
+#include <metal/metal.h>
+#include <metal/stddef.h>
+#include <metal/stdio.h>
+#include <metal/stdlib.h>
+#include <metal/string.h>
+#include "metalio.h"
+#include "qsam.h"
+#else
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#endif
+
+#include "zowetypes.h"
+#include "alloc.h"
+#include "bpxnet.h"
+#include "zos.h"
+#include "utils.h"
+#include "socketmgmt.h"
+
+#include "httpserver.h"
+#include "dataservice.h"
+#include "json.h"
+#include "datasetjson.h"
+#include "logging.h"
+#include "zssLogging.h"
+
+#include "datasetService.h"
+#include "datasetHeartbeatService.h"
+
+#ifdef __ZOWE_OS_ZOS
+
+static int serveDatasetHeartbeat(HttpService *service, HttpResponse *response){
+  time_t T;
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "begin %s\n", __FUNCTION__); 
```

Comment:
I doubt we want 'INFO' for these. Please don't forget to revert them if you set them for development.

Reply:
made it DEBUG - actually remove file just added function in already existing datasetService.c